.buildkite/commands/release-build.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash -eu
+
+# FIXIT-13.1: Temporary fix until we're on the Xcode 13.1 VM
+echo "--- :rubygems: Fixing Ruby Setup"
+gem install bundler
+
+echo "--- :arrow_down: Installing Release Dependencies"
+brew update # Update homebrew to temporarily fix a bintray issue
```

Comment:
There shouldn't be issues with Bintray anymore: https://github.com/Homebrew/discussions/discussions/691#discussioncomment-584299